staging/kos/pkg/registry/network/networkattachment/strategy.go

```diff
@@ -107,5 +109,29 @@ func (networkattachmentStrategy) Canonicalize(obj runtime.Object) {
 }
 
 func (networkattachmentStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
-	return field.ErrorList{}
+	var errs field.ErrorList
+	immutableFieldMsg := "attempt to update immutable field"
+	newNa, oldNa := obj.(*network.NetworkAttachment), old.(*network.NetworkAttachment)
+	if newNa.Spec.Node != oldNa.Spec.Node {
+		errs = append(errs, field.Forbidden(field.NewPath("spec.node"), immutableFieldMsg))
```

Comment:
It should be `field.NewPath("spec", "node")`

Reply:
Dang, fixed in 7690042.